Assets/Scripts/PlayerBase.cs

```diff
@@ -1,50 +1,53 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.InputSystem;
 
 public class PlayerBase : MonoBehaviour
 {
-    public InputManager inputManager;
-
-    private int playerNum;
     private Rigidbody2D rb;
     private bool grounded;
 
     public float horizontalMovementModifier;
     public float jumpVelocity;
-    public float fallMultiplier;
-    public float lowFallMultiplier;
+
+    private float xChange;
+    private bool jump;
```

Comment:
Nit: can this variable be named `isJumping` and the grounded variable be named `isGrounded`? I think it's clearer that they're booleans with that naming scheme.

Reply:
(optional) I've also seen a pattern in Unity docs where movement variables are named something like:

```
m_isJumping
m_xChange 
```

Reply:
That's a cool pattern! I'll refactor to include that 😄